onnxruntime/core/providers/common.h

```diff
@@ -69,51 +69,63 @@ inline AutoPadType StringToAutoPadType(const std::string& str) {
 }
 
 // helper function
+
 template <bool ForceSymmetricAutoPadding>
-Status ComputePadAndOutputShape(
-    const int64_t in_dim,
-    const int64_t stride,
-    const int64_t kernel,
-    const int64_t dilation,
-    AutoPadType pad_type,
-    int64_t* pad_head,
-    int64_t* pad_tail,
-    int64_t* out_dim) {
-  const int64_t dkernel = dilation * (kernel - 1) + 1;
+Status ComputePad(const int64_t in_dim,
```

Comment:
Can ForceSymmetricAutoPadding be a parameter instead?

Reply:
Will modify in next PR